unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/Sections/PlacesAndEventsSection/SubSections/PlacesSubSection/FavoritesSubSection/FavoriteSubSectionComponentView.cs

```diff
@@ -0,0 +1,410 @@
+    {
+        OnPlaceFavoriteChanged?.Invoke(placeId, isFavorite);
+    }
+
+    public void ShowPlaceModal(PlaceCardComponentModel placeModel)
+    {
+        placeModal.Show();
+        PlacesCardsConfigurator.Configure(placeModal, placeModel, null, OnPlaceJumpInClicked, OnVoteChanged, OnPlaceFavoriteChanged);
+    }
+
+    public void ShowWorldModal(PlaceCardComponentModel placeModel)
+    {
+        worldModal.Show();
+        PlacesCardsConfigurator.Configure(worldModal, placeModel, null, OnWorldJumpInClicked, OnVoteChanged, OnPlaceFavoriteChanged);
+    }
+
+    public void ShowAllPlaces(List<PlaceCardComponentModel> places, bool showMoreButton)
+    {
+        showMore.gameObject.SetActive(showMoreButton);
+        foreach (PlaceCardComponentModel placeCardComponentModel in places)
+        {
+            PlaceCardComponentView placeCardComponentView = fullPlacesPool.Get();
+            placeCardComponentView.model = placeCardComponentModel;
+            placeCardComponentView.RefreshControl();
+            placeCardComponentView.OnLoseFocus();
+            placeCardComponentView.transform.SetAsLastSibling();
+            pooledFullPlaces.Add(placeCardComponentView);
+            ConfigurePlaceCardActions(placeCardComponentView, placeCardComponentModel);
+        }
+        loadingAll.SetActive(false);
+        Utils.ForceRebuildLayoutImmediate(fullPlacesParent);
+    }
+
+    public void ShowAllWorlds(List<PlaceCardComponentModel> worlds, bool showMoreButton)
+    {
+        showMore.gameObject.SetActive(showMoreButton);
+        foreach (PlaceCardComponentModel placeCardComponentModel in worlds)
+        {
+            PlaceCardComponentView placeCardComponentView = fullWorldsPool.Get();
+            placeCardComponentView.model = placeCardComponentModel;
+            placeCardComponentView.RefreshControl();
+            placeCardComponentView.OnLoseFocus();
+            placeCardComponentView.transform.SetAsLastSibling();
+            pooledFullWorlds.Add(placeCardComponentView);
+            ConfigureWorldCardActions(placeCardComponentView, placeCardComponentModel);
+        }
+        loadingAll.SetActive(false);
+        Utils.ForceRebuildLayoutImmediate(fullWorldsParent);
+    }
+
+    private void InitializePools()
+    {
+        placesPool = new UnityObjectPool<PlaceCardComponentView>(placePrefab, placesParent);
+        placesPool.Prewarm(MAX_POOL_COUNT);
+        worldsPool = new UnityObjectPool<PlaceCardComponentView>(placePrefab, worldsParent);
+        worldsPool.Prewarm(MAX_POOL_COUNT);
+        fullPlacesPool = new UnityObjectPool<PlaceCardComponentView>(placePrefab, fullPlacesParent);
+        fullPlacesPool.Prewarm(MAX_POOL_COUNT);
+        fullWorldsPool = new UnityObjectPool<PlaceCardComponentView>(placePrefab, fullWorldsParent);
+        fullWorldsPool.Prewarm(MAX_POOL_COUNT);
+    }
+
+    public void RestartScrollViewPosition()
+    {
+    }
```

Comment:
**Question**: Could we remove this empty method?

Reply:
nope, because it's coming from IPlacesAndEventsSubSectionComponentView but in this case it's not needed